src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI/GenAPIApp.cs

```diff
@@ -35,7 +35,7 @@ public static void Run(ILog logger,
             bool resolveAssemblyReferences = assemblyReferences?.Length > 0;
 
             // Create, configure and execute the assembly loader.
-            AssemblySymbolLoader loader = new(resolveAssemblyReferences, respectInternals);
+            AssemblySymbolLoader loader = new(logger, resolveAssemblyReferences, respectInternals);
```

Comment:
As mentioned in the original PR, please rename this field to `log`. We use that name consistently in APICompat, i.e. https://github.com/dotnet/sdk/blob/535f8402195ba520ffe4671bd30b29b403b51c7d/src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/ValidateAssemblies.cs#L36

GenAPI was inconsistent in that regard as it was developed by different dev from outside our team and we didn't notice this during PR review.

Applies to all other changes in this PR as well.

Reply:
Ok, done.